web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -51,4 +51,8 @@ For example, to use GitHub actions as your CI service, you could add the followi
 
 `--ci=githubactions`
 
+Another example, to use Gitlab as your git host service & Gitlab CI/CD as your CI service, you could add the following additional flag to your `terminus build:project:create` command:
```

Comment:
Rather than having a full example for each maybe we could just collapse this down to one and provide the other options. So remove what was added here, and then at the start of the next section add something like: "Other possible values are circleci, gitlab-pipelines and bitbucket-pipelines."